src/Sylius/Bundle/ApiBundle/Command/Checkout/ShipShipment.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Checkout;
+
+use Sylius\Bundle\ApiBundle\Command\ShipmentIdAwareInterface;
+
+/** @experimental */
+class ShipShipment implements ShipmentIdAwareInterface
+{
+    /** @var string|null */
+    public $shipmentId;
+
+    /** @var string|null */
+    public $tracking;
```

Comment:
```suggestion
    public $trackingCode;
```
?

Reply:
I was also thinking about this, but before changes we were using `tracking` and in serialization groups `tracking` is also the way. Also with your suggestion we would make a BC break, I assume?

Reply:
I am not sure about BC break. word `tracking` IMO is misleading, and suggest boolean value instead of string with code. But I not have strong opinion :)

Reply:
I would be also for using `trackingCode` as it is more descriptive and we are using this name in UI. However, it would be a BC Break for API, so I suggest to add a note to UPGRADE file with such a change